docs(blog): add deployment action section to v1.30 blog (#5252)
aws/copilot-cli: site/content/blogs/release-v130.en.md

## Ctrl-C

## Deployment actions
Maybe you want to run a database migration before a workload deployment, and the workload's health check depends on the
update. Or maybe you'd like your pipeline to execute some end-to-end or integration tests after a workload deployment. These actions are

> now possible with [Copilot pipelines](../docs/concepts/pipelines.en.md)!  

Reviewer: This is just my personal thought maybe it's bad 🤣. I feel like "copilot" is implied by that this is a copilot blog post
```suggestion
now possible with [pipelines](../docs/concepts/pipelines.en.md)!  
```

Author: I added that bc before it was possible with other pipelines but not copilot pipelines... 🤷🏼

> While Copilot has supported 'test commands' for some time now, pre- and post-deployment actions extend pipeline functionality 

Reviewer: nit!
```suggestion
While Copilot has supported 'test_commands' for some time now, pre- and post-deployment actions extend pipeline functionality 
```
```suggestion
While Copilot has supported ['test_commands'](https://aws.github.io/copilot-cli/docs/manifest/pipeline/#stages-test-cmds) for some time now, pre- and post-deployment actions extend pipeline functionality 
```

and flexibility. For `test_commands`, Copilot inlines a buildspec with your command strings into the pipeline
Cloudformation template; for `pre_deployments` and `post_deployments`, Copilot reads the [buildspec(s)](https://docs.aws.amazon.com/codebuild/latest/userguide/build-spec-ref.html)
in your local workspace.  

You control all the configuration for these actions right in your [pipeline manifest](../docs/manifest/pipeline.en.md). You may have multiple pre-deployment actions and multiple 
post-deployment actions; for each one, you must provide the path to the [buildspec](https://docs.aws.amazon.com/codebuild/latest/userguide/build-spec-ref.html),
relative to your project root, in the `[pre_/post_]deployments.buildspec` field. Copilot will generate a CodeBuild project for your action, deployed

> in the same region as the pipeline and app. Use Copilot commands within your buildspec to access the VPC of the environment

Reviewer: ```suggestion
in the same region as the pipeline and app. Use Copilot commands - such as `copilot svc exec` or `copilot task run` within your buildspec to access the VPC of the environment
```

being deployed or deployed to; you may use the `$COPILOT_APPLICATION_NAME` and `$COPILOT_ENVIRONMENT_NAME` Copilot environment variables
to reuse your buildspec file for multiple environments.

> You can even specify the deployment order of actions within your pre-deployment and

Reviewer: ```suggestion
You can even specify the order of actions within your pre-deployment and
```

> post-deployment groups by using the `depends_on` field; by default, the actions will deploy in parallel. 

Reviewer: Not sure about "deploy", maybe "take place", "execute" or "run" 🤔 
```suggestion
post-deployment groups by using the `depends_on` field; by default, the actions will take place in parallel. 
```
